numpy/core/_add_newdocs.py

```diff
@@ -4163,11 +4166,11 @@ def luf(lamdaexpr, *args, **kwargs):
     --------
     >>> x = np.array([[0, 1], [2, 3]])
     >>> x.tobytes()
-    b'\\x00\\x00\\x00\\x00\\x01\\x00\\x00\\x00\\x02\\x00\\x00\\x00\\x03\\x00\\x00\\x00'
+    b'\\x00\\x00\\...'
     >>> x.tobytes('C') == x.tobytes()
     True
     >>> x.tobytes('F')
-    b'\\x00\\x00\\x00\\x00\\x02\\x00\\x00\\x00\\x01\\x00\\x00\\x00\\x03\\x00\\x00\\x00'
+    b'\\x00\\x00\\...'
```

Comment:
Again, this defeats the purpose of the test. I think the right fix here is to pass `dtype='<u2` above - it's probably failing due to the platform-specific nature of `np.dtype(int)`